src/appengine/libs/issue_management/issue_tracker_utils.py

```diff
@@ -177,6 +177,10 @@ def was_label_added(issue, label):
   if not label:
     return False
 
+  # Optimization that does not require pulling in issue's actions.
+  if any([label.lower() == l.lower() for l in issue.labels]):
```

Comment:
nit: no need for surrounding []. It'll make this faster too.

Reply:
done!